dandiapi/api/tasks.py

```diff
@@ -152,3 +154,33 @@ def validate_version_metadata(version_id: int) -> None:
     version.status = Version.Status.VALID
     version.validation_errors = []
     version.save()
+
+
+@shared_task
+def delete_doi(doi: str) -> None:
```

Comment:
Can you move this method to `doi.py` so all the DOI code is in one place? This task can simply be a wrapper around it.

Reply:
Done.